r-python-eds-lessons/README.md

```diff
@@ -0,0 +1,4 @@
+# r-python-eds-lessons
+
+Docker container with [Anaconda 3](http://continuum.io/downloads) and R for
+building the [Earth Data Science website](http://earthdatascience.org).
```

Comment:
suggestion: for url change  `[Earth Data Science website](https://www.earthdatascience.org)`